ADAL/src/telemetry/ADTelemetryDefaultEvent.m

```diff
@@ -171,10 +186,7 @@ + (NSDictionary*)defaultParameters
         }
     });
     
-    NSMutableDictionary *defaultParameters = [s_defaultParameters mutableCopy];
-    [defaultParameters adSetObjectIfNotNil:[ADIpAddressHelper adDeviceIpAddress] forKey:AD_TELEMETRY_KEY_DEVICE_IP_ADDRESS];
-    
-    return defaultParameters;
+    return s_defaultParameters;
```

Comment:
I see deviceId is still getting hashed in here and will be hashed second time in the setProperty.

`[s_defaultParameters adSetObjectIfNotNil:[deviceId adComputeSHA256] forKey:AD_TELEMETRY_KEY_DEVICE_ID];`

Reply:
It should not be hashed second time because it is different dictionary and we don’t call setProperty for it.

Reply:
nit: Is it possible for us to use setproperty for ipaddress? i just don't like hashing being called at different places.
I've not looked into the details on when setProperty is called, but would like to know if that is possible.